guardian/plutohelperagent
# Handle Premiere Pro Version Differences

## What does this change?

If a Premiere Pro version number is supplied in the URL, the software checks it against the versions installed locally. If one matches, that version is used to open the project. If none match, the software forwards the user to a URL on pluto-core, requesting the project be updated to the most recent version installed on the Mac.

Adds a Pluto base URL box in the preferences window which needs to be filled in for the access to the URL on pluto-core to work.

## How to test

Install the required changes to pluto-core. Run a build from GitHub of this software or build it. Add the base URL of your Pluto installation in the preferences window. Set up pluto-core in the Premiere Pro versions section with the required Premiere Pro versions. Remember that version 15 should be listed as '15.0.0' and version 14.9 should be listed as '14.9.0'. Attempt to open a project file from pluto-core. If the versions match it should attempt to open it locally. If not, you should get fowarded to a page on pluto-core offering a change to the version in the file.

## Images

![image](https://user-images.githubusercontent.com/10620802/161245975-33e353d6-86a4-4985-b2df-387de4fbf6d7.png)
